crates/subspace-runtime/src/lib.rs

```diff
@@ -121,7 +121,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: Cow::Borrowed("subspace"),
     impl_name: Cow::Borrowed("subspace"),
     authoring_version: 0,
-    spec_version: 2,
```

Comment:
We do not want to change the spec version unfortunately. 
Just pallet storage versions.

Reply:
Makes sense, the only remaining change is the EVM domain transaction version, what do you want done with that?

Reply:
Sorry, it wasn't clear to me what you meant.